frappe/desk/search.py

```diff
@@ -93,13 +94,26 @@ def search_widget(doctype, txt, query=None, searchfield=None, start=0,
 			order_by_based_on_meta = get_order_by(doctype, meta)
 			order_by = "if(_relevance, _relevance, 99999), {0}, `tab{1}`.idx desc".format(order_by_based_on_meta, doctype)
 
-			values = frappe.get_list(doctype,
-				filters=filters, fields=formatted_fields,
-				or_filters = or_filters, limit_start = start,
-				limit_page_length=page_length,
-				order_by=order_by,
-				ignore_permissions = True if doctype == "DocType" else False, # for dynamic links
-				as_list=not as_dict)
+			if doctype == "DocType":
```

Comment:
maybe just treat the parameters separately, this repetition seems (marginally) un-necessary 🤔

Reply:
I have made some changes, can you please let me know if that's more aligned with what you have in mind ?
Thank you!